aodndata/aims/common.py

```diff
@@ -0,0 +1,63 @@
+import re
+from datetime import datetime
+
+from netCDF4 import Dataset
+
+
+def get_main_var(filepath):
+    netcdf_file_obj = Dataset(filepath, mode='r')
```

Comment:
This could also be a `with Dataset(filepath, mode='r') as netcdf_file_obj:`, as you've done in most  of your other functions.